docs/developers/build-aoc.md

```diff
@@ -1,6 +1,6 @@
 ### Build Artifacts
```

Comment:
Should we include requirements such as install make, git, golang version instead of having independent packages?